### PR TITLE
Add client to fluentbit-operator

### PR DESCRIPTION
**Pull request by keladhruv**
**Context**
Currently the fluentbit-operator repo does not have a client which can be readily used. This PR adds a client to interact with the fluentbit-operator.

**Significant Changes**
There are the following significant changes that have been made as part of this PR-

- Adding a client using [code-generator ](https://github.com/kubernetes/code-generator)
- Moving the `/api/v1alpha2` folder to `/api/fluentbitoperator/v1alpha2` to make it compatible with the `client-gen` format.

### CODE REVIEW

File: api/fluentbitoperator/v1alpha2/groupversion_info.go [hunk 1]
```diff
@@ -24,11 +24,11 @@ import (
 )
 
 var (
-	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "logging.kubesphere.io", Version: "v1alpha2"}
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "logging.kubesphere.io", Version: "v1alpha2"}
```
Comment (keladhruv) [inline api/fluentbitoperator/v1alpha2/groupversion_info.go:28]: This change makes it compatible with the code generated by `client-gen`

**Comment by benjaminhuo** [comment]
@keladhruv This another great PR from digital ocean.
Thanks very much! 
We would like to invite you and @bojand as the maintainer of FluentBit Operator.

**Review by wanjunlei — APPROVED** [review]
I think this pr is very good. There are just two small doubts

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**